### ISSUE TITLE
Cardno: Fix population values per country from WorldPop

### ISSUE THREAD

**Issue by rossbernet** [geotrellis/geotrellis-road-distance-sdg#43]
as per the results of the investigation in #42, we realized that the current SDG workflow generates an incorrect country population summary for WorldPop based on the way it excludes certain tiles due to a lack of overlap with Natural Earth boundaries (I think?)

This task is to re-run the global job to generate the correct population summaries, which will require the regeneration of the global vectorTiles layer and country geojson file.

Ideally the tile locations can stay the same as per the [release doc](https://github.com/geotrellis/geotrellis-road-distance-sdg/releases) so that Jeff's URL's don't break in the app [visualization repo](https://github.com/azavea/rai-measurement-tool)

(We discussed the possible exclusion of Russia if it's problematic, Fiji too?)

**Comment by CloudNiner** [comment]
Not necessary to exclude RUS/FJI, there's [a hack in place](https://github.com/geotrellis/geotrellis-road-distance-sdg/blob/master/src/main/scala/geotrellis/sdg/Country.scala#L28-L34) that always tweaks their extents when we generate the image/vector pyramids.

**Comment by CloudNiner** [comment]
To regenerate tiles in place, you'll want to use:
```
sparkSubmitMain geotrellis.sdg.PopulationNearRoads \
  --output /cardno-countries.geojson \
  --tileLayerUriPrefix s3://un-sdg/tiles/cardno
```

Note that since this overwrites tiles, there's likely caching in place for any tiles you already pulled in your browser. To see the updated changes you'll need to manually reset cache for the site in your browser.

**Comment by echeipesh** [comment]
The corrected values are added as `rai-v3.0.2.geojson.zip` to existing release: https://github.com/geotrellis/geotrellis-road-distance-sdg/releases/tag/v3.0
